tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/clean_height_weight_test.py

```diff
@@ -388,7 +388,7 @@ def setUp(self):
                 {concept_ancestor_data_query}'''
         ])
```

Comment:
I wonder if these can be broken up to prevent them all running at the same time? I'm not sure if/how much it helps, but it seems to work for other tests running on the same table.
```suggestion
self.load_test_data([
            person_data_query, measurement_data_query,
            measurement_ext_data_query, condition_data_query,
            concept_data_query, concept_ancestor_data_query
         ])
```

Reply:
I split this into three different loads to reduce the number of queries running simultaneously. This is ready for another looksie.

Reply:
I see. I think best would be to send them as a list of six individual queries without the f-strings in a single call to `load_test_data`, so they all run separately. But this could work as well.

Reply:
@nishanthpp93 I have them as the separate queries now instead of the f-strings so this is ready for another quick review before I merge